web/gds-user-ui/src/modules/dashboard/member/utils.ts

```diff
@@ -87,3 +87,18 @@ export async function copyToClipboard(data: any) {
       console.error('[copyToClipboard]', err);
   }
 }
+
+export const getBusinessCategory = (category: any) => {
+  switch (category) {
+    case 1 || 'PRIVATE_ORGANIZATION':
+      return t`Private Organization`;
+    case 2 || 'GOVERNMENT_ENTITY':
+      return t`Government Entity`;
+    case 3 || 'BUSINESS_ENTITY':
+      return t`Business Entity`;
+    case 4 || 'NON_COMMERCIAL_ENTITY':
+      return t`Non-Commercial Entity`;
+    default:
+      return t`Unknown Entity`;
+  }
+};
```

Comment:
```suggestion
export const getBusinessCategory = (category: any) => {
  switch (category) {
    case 1:
    case 'PRIVATE_ORGANIZATION':
      return t`Private Organization`;
    case 2:
    case 'GOVERNMENT_ENTITY':
      return t`Government Entity`;
    case 3:
    case 'BUSINESS_ENTITY':
      return t`Business Entity`;
    case 4:
    case 'NON_COMMERCIAL_ENTITY':
      return t`Non-Commercial Entity`;
    default:
      return t`Unknown Entity`;
  }
};
```

This ensures that both the numeric value and the corresponding string representation will be handled correctly.